account_invoice_margin/hook.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from odoo.tools.sql import column_exists, create_column
```

Comment:
A little bit simpler

```suggestion
from odoo.tools import column_exists, create_column
```

Reply:
Done